arangod/Graph/Graph.cpp

```diff
@@ -116,6 +116,20 @@ Graph::Graph(velocypack::Slice const& slice, ServerDefaults const& serverDefault
   TRI_ASSERT(!_rev.empty());
 
   if (slice.hasKey(StaticStrings::GraphEdgeDefinitions)) {
+    // TODO: Check why we're landing here and not in SmartGraphEE - Cleanup!
+    LOG_DEVEL << "do not forget me - 1";
+    if (slice.isObject()) {
```

Comment:
This if looks like it is in the wrong position.
I would have expected it on top level, before the check for GraphEdgeDefinitions.